lib/promisifier.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+
+/**
+ * Creates promisifier.
+ *
+ * @param {bluebird} bluebird
+ * @return {promisifier}
+ */
+module.exports = (bluebird) => {
+    return (otherModule) => {
+        if (typeof otherModule === "function") {
+            otherModule = bluebird.promisify(otherModule);
+        }
+
+        return bluebird.promisifyAll(otherModule);
+    };
+};
```

Comment:
What do you think about making a `dizzy-bluebird` module that does this for you?  I don't know if I like seeing the same code in three projects.

Perhaps not for merge request #1, but at least something we can drop in Jira and pick up if we have a spare hour?  The projects that need it would be DDQ, DDQ-backend-mysql, OpenToken and now OpenToken Lib JS.